docs/src/app/components/pages/components/DropDownMenu/ExampleSimple.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import DropDownMenu from 'material-ui/lib/DropDownMenu';
+import MenuItem from 'material-ui/lib/menus/menu-item';
+
+export default class DropDownMenuSimpleExample extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {value: 2};
+  }
+
+  handleChange = (e, index, value) => this.setState({value});
+
+  render() {
+    return (
+      <DropDownMenu value={this.state.value} onChange={this.handleChange}>
```

Comment:
Have a look here https://facebook.github.io/react/docs/forms.html#uncontrolled-components.
It's well explained :smile: